### DISCUSSION TITLE
tiddler 中到本地目录的链接无法打开，而 tiddlydesktop 则无此问题

### DISCUSSION THREAD

**Discussion by dodoowang** [tiddly-gittly/TidGi-Desktop#241] [Q&A]
链接形如：file:///d:/MyDocs

**Comment by linonetwo** [comment]
应该要用 [ext[file:///]] 的形式，试试看？

**Reply by dodoowang** [reply]
还是不行，表现一样

**Reply by linonetwo** [reply]
之前没加这个功能，下个版本带上，可以 watch 关注一下更新。详见 https://tw-cn.netlify.app/#%E6%89%93%E5%BC%80%E5%A4%96%E9%83%A8%E6%96%87%E4%BB%B6

**Reply by dodoowang** [reply]
用 0.7.7 版试了一下，通过 `<a href={{!!path}}>{{!!name}}</a>` 方式添加的链接（path 和 name 中分别是本地文件的 uri 和文件名），在链接上按右键选择在浏览器打开，如果 path 中不包含中文，是可以正常打开的，否则就会报错：

Unhandled Promise Rejection
Error: Failed to open: ϵͳ�Ҳ���ָ�����ļ��� (0x2)

期待新版本
